nixos/modules/services/networking/networkmanager.nix

```diff
@@ -229,20 +229,36 @@ in {
             source = mkOption {
               type = types.path;
               description = ''
-                A script.
+                Path to the hook script.
               '';
             };
 
             type = mkOption {
               type = types.enum (attrNames dispatcherTypesSubdirMap);
               default = "basic";
               description = ''
-                Dispatcher hook type. Only basic hooks are currently available.
+                Dispatcher hook type. Look up the hooks described at
+                https://developer.gnome.org/NetworkManager/stable/NetworkManager.html
```

Comment:
One more, please. Change this to `<xlink>`, otherwise it isn't rendered as link
```
<link xlink:href="https://developer.gnome.org/NetworkManager/stable/NetworkManager.html">https://developer.gnome.org/NetworkManager/stable/NetworkManager.html</link>
```

Reply:
done